content/source/docs/cloud/vcs/index.html.md

```diff
@@ -46,7 +46,7 @@ To use configurations from VCS, Terraform Cloud needs to do several things:
 Terraform Cloud uses webhooks to monitor new commits and pull requests.
 
 - When someone adds new commits to a branch, any Terraform Cloud workspaces based on that branch will begin a Terraform run. Usually a user must inspect the plan output and approve an apply, but you can also enable automatic applies on a per-workspace basis. You can prevent automatic runs by locking a workspace.
-- When someone submits a pull request/merge request to a branch from another branch in the same repository, Terraform Cloud performs a [speculative plan](../run/index.html#speculative-plans) with the contents of the request and links to the results on the PR's page. This helps you avoid merging PRs that cause plan failures.
+- When someone submits a pull request/merge request to a branch, any Terraform Cloud workspaces based on that branch will perform a [speculative plan](../run/index.html#speculative-plans) with the contents of the request and links to the results on the PR's page. This helps you avoid merging PRs that cause plan failures.
```

Comment:
TFC's going to continue ignoring other forks, right? So with these removals, you're figuring we probably only need to point that out once, in the .../run/ui.html page?

I suppose we can give it a shot; it probably won't end up freaking anyone out. :) If it does, we can always throw it back in.

Reply:
Yeah, that was my thinking. We weren't really consistent about whether or not we explicitly said branch in the same repository, just branch, or even just pull request in the various places where we talk about how this works, so I thought it would be safe to be more general here (was trying to avoid needing to add a whole bunch extra to call out that pull requests from forks are possible in every place we mention this flow).